js/block.js

```diff
@@ -7398,9 +7392,15 @@ function Block(protoblock, blocks, overrideName) {
 
         // Update the block value and block text.
         if (this.name === "number") {
+            var cblk1 = this.connections[0];
+            var cblk2 = this.blocks.blockList[cblk1].connections[0];
```

Comment:
Why this check?

Reply:
I meant why cblk2 -- the connection to the connection. But I assume that is because there may be a divide block in between -- the usual situation. That is probably adequate.